_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme

```diff
@@ -396,6 +435,11 @@ function rhdp2_preprocess_node(&$variables) {
 
       $variables['timestamp'] = $timestamp;
     }
+    
+    // Prepares the article authors render element for taxonomy page.
```

Comment:
There are 3 calls to _generate_authors_links split up across the file here, I have to jump to each 'if content type'
 check to read this.

I would rather see this collected into one code block,  for example line 151 for similar pattern
`if (in_array($type, $measured_read_time_types) && $variables['view_mode'] == 'learning_path_card')`
so that I don't have to jump to each content-type check to read the same code.

Also, please check PHPCS, "Found 7 errors and 1 warning" for rhdp2.theme.

Reply:
@staceymosier Resolved in 3c243f5